### DISCUSSION TITLE
Any reason why github release names are not included in the release notes?

### DISCUSSION THREAD

**Discussion by HenrikPoulsen** [renovatebot/renovate#18689] [Request Help] [answered]
If you look in `lib/workers/repository/update/pr/changelog/release-notes.ts`
you see `if (releaseNotes.body.length) {`, if the body is empty then the release notes are ignored even if there is a name set.

Followed by the `lib/workers/repository/update/pr/changelog/hbs-template.ts` not making use of `release.releaseNotes.name` when it exists.

If you were to change `release-notes.ts` to instead do:
`if (releaseNotes.body.length || releaseNotes.name?.length) {`

followed by something like changing the release heading in the release notes from:
``### [`v{{{release.version}}}`]({{{release.releaseNotes.url}}})``
to
``### [`v{{{release.version}}}`]({{{release.releaseNotes.url}}}){{#if release.releaseNotes.name}}: {{release.releaseNotes.name}}{{/if}}``
then you would expose this missing information.
Is there a reason for not having it like this or is it something that has just been missed and should be added?

before:

![image](https://user-images.githubusercontent.com/472627/199219670-a28d6d5b-4bba-4615-b5d8-77a78cdabf5a.png)
after:

![image](https://user-images.githubusercontent.com/472627/199219441-d376d61f-732e-4740-a066-e1e63c525e3a.png)

**Comment by rarkins** [comment, marked as answer]
No reason, and PRs welcome

**Reply by HenrikPoulsen** [reply]
Thanks I'll get my changes ready for upstream
